jwst/assign_wcs/util.py

```diff
@@ -1243,3 +1246,177 @@ def in_ifu_slice(slice_wcs, ra, dec, lam):
     onslice_ind = np.isclose(slx, SLX,atol=5e-4)
 
     return onslice_ind
+
+
+def update_fits_wcsinfo(datamodel, max_pix_error=0.01, degree=None, npoints=32,
+                        crpix=None, projection='TAN', **kwargs):
+    """
+    For imaging data models *only*, update data model's ``meta.wcsinfo``
+    attribute using a FITS SIP approximation of the current data model's
+    GWCS from ``meta.wcs``.
+
+    The default mode in using this attempts to achieve roughly 0.01 pixel
+    accuracy over the entire image.
+
+    This function uses on the :py:meth:`~gwcs.wcs.WCS.to_fits_sip` to
```

Comment:
"uses on the" - is this a typo?  Should "on" be removed?

Reply:
yes, give me 1 min. I'll fix the rest of the comments